### PR TITLE
Internationalisation and tool to parse surface forms from Spotlight Index

### PR DESCRIPTION
**Pull request by JulienCojan**
Properties moved to an external file, including langCode, resourceNamesSpace, in order to work with i18n chapters.

Now run the server with mvn scala:run -Dlauncher=Server -DaddArgs=indexing.properties

The class org.dbpedia.lookup.util.CreateLookupIndex parses the surface form generated by Spotlight Indexer to generate the rdf file needed for Lookup Indexer.
It is called by script createIndexFromSpotlightIndex.sh

### CODE REVIEW

File: src/main/scala/org/dbpedia/lookup/server/Result.scala [hunk 1]
```diff
@@ -20,7 +20,7 @@ class Result(uri: String,
              refCount: Int) {
 
     // get the label of this result URI
-    def label = WikiUtil.wikiDecode(uri.replace("http://dbpedia.org/resource/", ""))
+    def label = WikiUtil.wikiDecode(uri.replace("http://fr.dbpedia.org/resource/", ""))
```
Comment (sandroacoelho) [inline src/main/scala/org/dbpedia/lookup/server/Result.scala]: Hardcoded URI  // We would like to receive it from a properties file

File: src/main/scala/org/dbpedia/lookup/server/Result.scala [hunk 2]
```diff
@@ -33,7 +33,7 @@ class Result(uri: String,
 
     // make a category label from a category URI
     def categoryLabel(categoryUri: String): String = {
-        WikiUtil.wikiDecode(categoryUri.replace("http://dbpedia.org/resource/Category:", ""))
+        WikiUtil.wikiDecode(categoryUri.replace("http://fr.dbpedia.org/resource/Catégorie:", ""))
```
Comment (sandroacoelho) [inline src/main/scala/org/dbpedia/lookup/server/Result.scala]: Hardcoded URI  // We would like to receive it from a properties file

Comment (mullekay) [inline src/main/scala/org/dbpedia/lookup/server/Result.scala]: Not sure if hard-coding prefixes is the way to go? Would it not be
better to get prefixes like this from a config parameter?

Am 04.02.2016 um 09:13 schrieb Sandro:

**Comment by sandroacoelho** [comment]
Salut @JulienCojan.

My apologies for the big delay and thank you for contributing with your time and code. I've checked your PR and some i18n are not completely solved.

Would you like to review it and send a PR again?

Thank you again

File: src/main/scala/org/dbpedia/lookup/server/Result.scala (hunk 2 above)
Comment (sandroacoelho) [inline src/main/scala/org/dbpedia/lookup/server/Result.scala]: Hi @mullekay .  In order to have a complete i18n support , is a good idea to have it out of the code

**Pull request closed without merging**